plugin/core/types.py

```diff
@@ -305,6 +305,8 @@ def r(name: str, default: Union[bool, int, str, list, dict]) -> None:
     def highlight_style_region_flags(self, style_str: str) -> Tuple[int, int]:
         if style_str in ("background", "fill"):  # Backwards-compatible with "fill"
             return sublime.DRAW_NO_OUTLINE, sublime.DRAW_NO_OUTLINE
+        elif style_str == "outline":
+            return sublime.DRAW_NO_FILL, sublime.DRAW_OUTLINED | sublime.DRAW_NO_FILL
```

Comment:
![deprecated](https://user-images.githubusercontent.com/6579999/230283349-38ecb119-84e1-4cb6-8ae8-ba5f319df6f3.png)

I think we should follow that suggestion. Also `32 | 32` is just `32`, so you can omit the first part `sublime.DRAW_OUTLINED | ` of the second return argument.

Should probably also update the settings file with the new value
https://github.com/sublimelsp/LSP/blob/43d00544f988f4a415f8fc9569436ff4eb7764d3/LSP.sublime-settings#L169